style.css

```diff
@@ -1 +1,366 @@
-/*  Add your styling here */
+body {
+    display: grid;
+    margin: 0 auto;
+    grid-template-columns: 1rem auto 0.5rem 0.5rem auto 1rem;
+    grid-template-rows: 130px 1rem 190px 1rem 200px 1rem 160px 1rem 160px 1rem 150px;
+    grid-template-areas: "header  header header header header header" ". . . . . ." ". welcome welcome welcome welcome ." ". . . . . ." ". hero  hero  hero  hero ." ". . . . . ." ". birthday . . cupcakes ." ". . . . . ." ". wedding  . . gluten ." ". . . . . ." "footer footer footer footer footer footer";
+    margin: 0;
```

Comment:
nice job on using CSS grid! I would suggest creating a class within `body` on which to apply the `display: grid` property, as it gives you more control over what should be part of the grid layout.
Also can you think of a way to use `column-gap` / `row-gap` instead of creating empty rows and columns? Have a look at the docs [here](https://css-tricks.com/snippets/css/complete-guide-grid/#column-gaprow-gapgrid-column-gapgrid-row-gap), or we can go through it in our session on Wednesday